### PR TITLE
Add Image Classification blog post

### PR DESCRIPTION
**Pull request by dpshekhawat**
This PR contains the blog post for the Flotta Image Classification Tool.

### CODE REVIEW

File: _posts/2022-08-27-image-classification-on-edgedevices.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- `THRES = 0.25` _confidence threshold currently 25%_
+- `TIMEINT = 5` _timeinterval for capturing every 5 secs_
+- `CAPTURE = True` _start capturing if True_
+
+The default directory `"images/"` is used for saving images.
+
+
+## Workflow 
+The workload is first deployed on the edgedevice once it has been configured using Flotta operator. 
+( Checkout how to configure device with flotta [here](https://project-flotta.io/flotta/2022/04/15/flotta-and-raspberry-pi.html#installation) )
```
Comment (pkliczewski) [inline _posts/2022-08-27-image-classification-on-edgedevices.markdown:39]: I think we have device configuration in `Prerequisite` so no need to have it here.

**Comment by dpshekhawat** [comment]
Before publishing the blog, I will change some link addresses to redirect to the project-flotta repo.

**Comment by dpshekhawat** [comment]
The Workflow section should be shown before getting_started or its okay to be at the end. What's your view on this?

**Review by pkliczewski — COMMENTED** [review]
Looks good, please let me know when ready

**Review by gciavarrini — COMMENTED** [review]
A clear and effective blog post: good job!

File: _posts/2022-08-27-image-classification-on-edgedevices.markdown [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+```shell
+#video group access to flotta user 
+[root@device ~]# usermod -a -G video flotta
+
+#expected
+[root@device ~]# id flotta
+uid=1001(flotta) gid=1001(flotta) groups=1001(flotta),39(video)
+```
+
+Following device registration, to run our workload on specific device, we need to label the device -
```
[the remark this answers is not in the document]
Comment (dpshekhawat) [reply _posts/2022-08-27-image-classification-on-edgedevices.markdown]: added, please check again.

File: _posts/2022-08-27-image-classification-on-edgedevices.markdown [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+## Prerequisite
+- Flotta-configured device. (see [documentation](https://project-flotta.io/documentation/v0_2_0/intro/overview.html))
+
+## Getting Started
+
+**Project repository can be found [here](https://github.com/dpshekhawat/image-classification).**
+
+### Preparations
+As part of the flotta installation, prioritizing security of containers we create the user flotta and utilise that user to execute the tasks.
+By default, the flotta user does not have access to any mounted devices, thus we must first make changes to provide flotta user access to the device in /dev before configuring the runtime to permit the access.
```
[the remark this answers is not in the document]
Comment (dpshekhawat) [reply _posts/2022-08-27-image-classification-on-edgedevices.markdown]: sure,

File: _posts/2022-08-27-image-classification-on-edgedevices.markdown [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+| THRES | Boxes with a confidence score less than confidence threshold are discarded  | float | >=0.25 |
+| TIMEINT | time interval for detection | int | 5 |
+
+
+## Workflow 
+![Workflow](/assets/images/flotta_image_classification-workflow.png)
+
+Once the edge device is configured, it begins real-time object recognition on the camera feed at the specified time interval and if an object is detected it captures and stores the images with the localization of those objects. 
+
+The functionality of the Flotta agent and Operator of bidirectional data sync between the device and storage is then used to sync these images from the device with object storage. (learn more on Data Synchronization [here](https://project-flotta.io/documentation/v0_2_0/operations/data_synchronization.html))
```
[the remark this answers is not in the document]
Comment (dpshekhawat) [reply _posts/2022-08-27-image-classification-on-edgedevices.markdown]: done!

**Review by pkliczewski — APPROVED** [review]

**Pull request merged**